README.md

```diff
@@ -166,6 +166,20 @@ also, with this flag to false, Bash-it will not show the repository as dirty whe
 
 **NOTE:** If you set in git configuration file the option to ignore *untracked* files, this flag has no effect, and Bash-it will ignore *untracked* files always.
 
+#### Git user
+
+In some environments it is useful to know the value of the currently-set git user, which is used to mark all new commits. For example, any organization that uses the practice of pair programming will typically author each commit with a [combined name of two authors](https://github.com/pivotal/git_scripts). When another pair uses the same pairing station, the authors must be changed.
+
+To enable display of the current pair in the prompt, you can set `SCM_GIT_SHOW_CURRENT_USER` to `true`. Once set, the `SCM_CURRENT_USER` variable will be automatically set to the initials of the git author(s). It will also be included in the default git prompt.
+
```

Comment:
Ah, very interesting, I sort of expected somehow that `git config --get user.initials` always returns something, but perhaps it only does so in the context of git pair, when it's explicitly set beforehand?

My intention was that in this use case, when the user is not using git pair, it just falls back to the current's user initials. What do you think about this: if `user.initials` are not set, we extract user initials from the `user.name`, as follows:

```bash
git config --get user.name | sed -e 's/ /\n/g;s/^\([^ ]\).*/\1/gm;' | tr 'A-Z' 'a-z' | tr '\n' ',' | sed 's/,//g'
```

In my shell this transforms 'Konstantin Gredeskoul' into 'kg'. That way, it would work in both cases?